components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -195,7 +194,7 @@ - (void)testTopHeaderHeightWithHeader {
 
   // Then
   XCTAssertEqualWithAccuracy(self.fakeBottomDrawer.topHeaderHeight,
-                             mdcDeviceTopSafeArea + fakePreferredContentSize.height, 0.001);
+                             MDCDeviceTopSafeAreaInset() + fakePreferredContentSize.height, 0.001);
```

Comment:
This seems unrelated to the screenshot tests.
lets put it in a different CL

Reply:
This change was originally needed because the snapshot tests require an app host and bundling the snapshot tests with the rest of the unit tests meant all our tests were running with an app host, so this test was failing on simulators with a notch.

I will move away from this and instead have the snapshot tests run in their own target. So I'll undo this change.